dev_requirements.txt

```
@@ -6,6 +6,8 @@ pyquil~=3.5.4
 pennylane-qiskit~=0.33.1
 pennylane~=0.33.1
 amazon-braket-sdk~=1.65.1
+qibo~=0.1.11
```

@francescsabater this should fix the problem with the Windows tests, as it seems the Windows dependency resolver behaves differently from Mac and Linux.

```suggestion
qibo==0.1.11
```

---

Yes, problem fixed! Thanks  @Misty-W  and @bdg221 @purva-thakre !!!